jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/wizard/SamProjectGenerator.kt

```diff
@@ -30,12 +30,14 @@ import software.aws.toolkits.resources.message
 import javax.swing.Icon
 import javax.swing.JComponent
 
-// ref: https://github.com/JetBrains/intellij-plugins/blob/master/vuejs/src/org/jetbrains/vuejs/cli/VueCliProjectGenerator.kt
+/**
+ * [ProjectTemplate] To allow for us to shim it into ProjectTemplatesFactory and use this in IntelliJ
```

Comment:
`DirectoryProjectGeneratorBase` so this shows up in light wizards?

Reply:
Good call